db/migrations/2008040105_rework_root_group.sql

```diff
@@ -0,0 +1,13 @@
+-- +migrate Up
+DELETE FROM `groups` WHERE `type`='Base' AND `text_id`='Root';
```

Comment:
In the anonymized db, there were no permission granted to this group?

Reply:
There were some, but they disappeared completely after the DB migration

Reply:
> There were some, but they disappeared completely after the DB migration

Sorry. This was a wrong statement. Actually there are 80 rows in permissions_granted, all with the 'content' view permissions (probably, first time I was checking after removal of the root group). So, these 80 permissions should be copied for the 'AllUsers' group.

Reply:
Fixed